### ISSUE TITLE
add a readme?

### ISSUE THREAD

**Issue by jpvajda** [BoD/gh-actions-tests#1]
@BoD Could you add a simple readme about this repo and it's purpose?

**Comment by BoD** [comment]
Haha well this is just me testing things! It's a throwaway repo 😊.Description should be "Nothing to see here, move along" 😅

**Comment by jpvajda** [comment]
@BoD oh no worries then! I though this repo might be used by others to test our Kotlin client. If you feel that isn't the case a readme isn't necessary. I'd be interested in how you are using this repo though for my own knowledge so if you want to share that I'd be appreciative.

**Comment by BoD** [comment]
Yes totally 👍 it was just a way to execute GitHub actions without creating 'noise' to Apollo's repo 😊 I didn't know about https://github.com/nektos/act to test actions locally, which is probably a nicer way to do that!
